.travis.yml

```diff
@@ -22,5 +22,7 @@ matrix:
         - curl -H 'Cache-Control:no-cache' https://raw.githubusercontent.com/fossas/fossa-cli/master/install.sh | BINDIR=$(pwd) bash
       script:
         - ./fossa # analyze dependencies and upload them (fails when missing api key)
+      after_success:
+        - ./fossa test
```

Comment:
Questions:
1. what does this `fossa test` do?
2. why run `after_success` ?

Reply:
I've updated the PR description which would cover these points

Reply:
Don't we want the build to fail if `fossa test` fails? If yes, we might want to move this to run as a 2nd command in `script` instead of `after_success`.

"The exit code of after_success, after_failure, after_script, after_deploy and subsequent stages do not affect the build result. However, if one of these stages times out, the build is marked as failed."
https://docs.travis-ci.com/user/job-lifecycle/

Reply:
Yes, I think Sarmad makes a good point.  Note that the overall matrix build will not fail if TASK=fossa fails due to https://github.com/zendesk/zendesk_app_framework_sdk/pull/96/files#diff-354f30a63fb0907d4ad57269548329e3R28 , but we do need to know if the fossa task fails.

Reply:
Hmm strange, Fossa doc recommends exactly this (using `after_success` to fail the build) https://docs.fossa.com/docs/travisci#section-blocking-ci-builds-w-fossa-issue-status. Let me give it a quick test.

Reply:
I verified that exit code is ignored in `after_success`, changed the code to run `fossa test` in `script` now.